.circleci/config.yml

```diff
@@ -7,15 +7,17 @@ defaults: &defaults
     - &restore_gems
       restore_cache:
         keys:
-          - gems-{{ checksum "Gemfile" }}-{{ checksum "Appraisals"}}-{{ .Environment.CIRCLE_JOB }}
+          - v2-gems-{{ checksum "Gemfile" }}-{{ checksum "Appraisals"}}-{{ .Environment.CIRCLE_JOB }}
+          - v2-gems-{{ checksum "Gemfile" }}
```

Comment:
An explanation for the version prefix would be useful, if only for posterity.

Reply:
Absolutely, I've added it in the commit description 😉 
> prefix cache key with version to allow busting cache

Reply:
There is no way (or at least an obvious way) to clear the cache created with safe_cache in circleci. e.g. clearing the cache from the settings is not doing the trick, running the job with the "without cache" will also fail to clear it.

Unfortunately I've managed to cache an empty directory for `gems-{{ checksum "Gemfile" }}-{{ checksum "Appraisals"}}-{{ .Environment.CIRCLE_JOB }}` and.. most probably this is the reason for the change.

Reply:
Thanks, both. 🙇